### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
JS2 week3 homework completed

### CODE REVIEW

File: javascript/javascript2/week3/fastestpresser.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        lCountWin.innerHTML = ""; 
+        sCountWin.innerHTML = "";
+    }
+    if(event.code === "KeyS"){
+        sCount++;
+        sCountNumber.innerHTML = sCount;
+        lCountWin.innerHTML = ""; 
+        sCountWin.innerHTML = "";
+    }
+}
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/fastestpresser.js:26]: Format your code consistently.
```suggestion
    if (timeDuration.value > 0) {
        if(event.code === "KeyL"){
            lCount++;
            lCountNumber.innerHTML = lCount;
            lCountWin.innerHTML = ""; 
            sCountWin.innerHTML = "";
        }
        if(event.code === "KeyS"){
            sCount++;
            sCountNumber.innerHTML = sCount;
            lCountWin.innerHTML = ""; 
            sCountWin.innerHTML = "";
        }
    }
```

File: javascript/javascript2/week3/fastestpresser.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+gameForm.addEventListener("click",(event)=>{
+    if (timeDuration.value > 0) {
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/fastestpresser.js:29]: To reduce code indentation, you can return early.
```suggestion
    if (timeDuration.value <= 0) {
        return;
    }
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week3/fastestpresser.js (hunk 1 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript2/week3/fastestpresser.js]: Changes done.

File: javascript/javascript2/week3/fastestpresser.js (hunk 2 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript2/week3/fastestpresser.js]: Changes done.

**Pull request merged**